x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/action/EsqlResponseListener.java

```diff
@@ -27,14 +30,21 @@ public class EsqlResponseListener extends RestResponseListener<EsqlQueryResponse
     private final RestChannel channel;
     private final RestRequest restRequest;
     private final MediaType mediaType;
+    /**
+     * Keep the initial query for logging purposes.
+      */
+    private final String esqlQuery;
     private final long startNanos = System.nanoTime();
     private static final String HEADER_NAME_TOOK_NANOS = "Took-nanos";
 
+    private static final Logger LOGGER = LogManager.getLogger(EsqlResponseListener.class);
```

Comment:
It's best to stick all of the `static` things above the non-static members. Or, at least, it's more traditional. That way you can scan the list of know what goes where.